src/Tribe/Views/V2/View.php

```diff
@@ -2194,23 +2194,54 @@ protected function should_show_recent_past_events_view() {
 	 */
 	protected function setup_additional_views( array $events = [], array $template_vars = [] ) {
 
-		$manager = tribe( Manager::class );
+		$manager      = tribe( Manager::class );
 		$default_slug = $manager->get_default_view_option();
 
-		// Only recent Past Events to only the default view.
+		// Show Recent Past Events only on the default view.
 		if ( $this->get_slug() !== $default_slug ) {
 			return;
 		}
 
+		$now    = $this->context->get( 'now' );
```

Comment:
Use a default value here to ensure we get something no matter what.

```suggestion
		$now    = $this->context->get( 'now', time() );
```